hoomd/md/ForceComposite.cc

```diff
@@ -799,7 +804,7 @@ void ForceComposite::computeForces(uint64_t timestep)
                 h_torque.data[central_idx].z += delta_torque.z;
 
                 /* from previous rigid body implementation: Access Torque elements from a single
-                   particle. Right now I will am assuming that the particle and rigid body reference
+                   particle. Right now I will be assuming that the particle and rigid body reference
```

Comment:
I think the whole "Right now..." sentence can be removed. The particle and body reference are the same. If this code was incorrect, then validation tests would fail.